doc/deployment/deploy_custom.md

```diff
@@ -1,6 +1,118 @@
 # Custom Deployments
 
-This document  details the various options of the `pachctl deploy custom ...` command for an on-premises deployment. 
+If you're deploying to a cloud infrastructure, 
+such as [Amazon Web Services (AWS)](https://pachyderm.readthedocs.io/en/latest/deployment/amazon_web_services.html),
+[Google Cloud Platform (GCP)](https://pachyderm.readthedocs.io/en/latest/deployment/google_cloud_platform.html), or 
+[Microsoft Azure](https://pachyderm.readthedocs.io/en/latest/deployment/azure.html), 
+Pachyderm's `pachctl deploy` includes commands for those providers (`amazon`, `google`, and `microsoft`, respectively).
+You should use those commands when deploying to those cloud infrastructures.
+You can customize cloud provider deployments extensively through flags available for each provider.
+
+Pachyderm also includes `pachctl deploy custom` for creating customized deployments for cloud providers or on-premises use.
+It's usually invoked with the `--dry-run` flag, 
+with the command's standard output directed into a series of customization scripts or a file for editing.
+
+This document covers using `pachctl deploy custom ... --dry-run` to create a manifest for a custom, on-premises deployment.
+We won't directly address automating your deployment by means of customization scripts,
+but we do encourage you to treat your [infrastructure as code](./on-premises.html#infrastructure-as-code).
+
```

Comment:
Before starting to break down the manifest that is generated, we probably need to break down the required and optional flags in deploy custom. You cover some of them in later sections in terms of how they populate the manifest, but we should probably upfront tell a user how to run the command. 

Also, there should likely be an example command and corresponding manifest (or at last snippets of the manifest).

Reply:
You're absolutely right.  I need to first narrow the choices to the set of deployment commands likely to be used on-premises.  I don't think that was "google", as the old doc had, but more likely AWS arguments.

When planning this documentation, we wanted the followon to be a set of PR's revamping the user experience for on-premises deployments. I'm coming to the conclusion that the user experiences for cloud vs on-premises deployments are substantially different.